AirCasting/Settings/SettingsView.swift

```diff
@@ -158,8 +158,21 @@ struct SettingsView: View {
 
     private var signOutLink: some View {
         NavigationLink(destination: SettingsMyAccountView(viewModel: SettingsMyAccountViewModel())) {
+            myAccount
+        }
+    }
+    
+    private var usernameText: some View {
+        Text(viewModel.username)
+            .font(Fonts.muliHeading2)
+            .foregroundColor(.aircastingGray)
+    }
+    
+    private var myAccount: some View {
+        VStack(alignment: .leading) {
             Text(Strings.Settings.myAccount)
                 .font(Fonts.boldHeading1)
```

Comment:
Would be nice if there was some more space between those texts

Reply:
Will that be okay?
![Screenshot 2022-06-28 at 14 44 41](https://user-images.githubusercontent.com/82096200/176181607-21a7550e-d27d-4b25-ac1f-35eb7df18ead.png)